ideascube/conf/base.py

```diff
@@ -252,7 +252,6 @@
     {
         'name': 'kalite',
         'description': _('Daemon which provides KhanAcademy on lan'),
-        'status_caller': lambda x: {'status': False} if subprocess.call(['pgrep', 'kalite', '-f']) else {'status': True}  # noqa
```

Comment:
Sure.

I figured that choosing which services we manage was orthogonal to this rework of how we managed, and it could come in another pull-request. :smile: